README.md

```diff
@@ -162,6 +162,34 @@ Old scheduled calls are deleted after 24 hours. You can cleanup the table by run
 npm run cleandb
 ```
 
+## Applying database migrations
+
+Database migrations are managed with [db-migrate](https://github.com/db-migrate/node-db-migrate). To create a new migration
+
+```bash
+npm run dbmigrate create description-for-your-migration
```

Comment:
Is there a format we should follow or does `db-migrate` do this for us? Like the number at the beginning?

Reply:
There's no defined naming convention to follow but generally I would go with naming it in a way that describes the migration. For example add-column-name-to-table-name, create-table-name, drop-table-name, add-unique-constraint-to-table-name